modules/Xsens_module/src/XsensRetargeting.cpp

```diff
@@ -258,3 +254,43 @@ bool XsensRetargeting::close()
 {
     return true;
 }
+bool XsensRetargeting::impl::mapJointsHDE2CONTROLLER(std::vector<std::string> robotJointsListNames,
+                                                     std::vector<std::string> humanJointsListName,
+                                                     std::vector<unsigned>& humanToRobotMap)
+{
+    if (!humanToRobotMap.empty())
+    {
+        humanToRobotMap.clear();
+    }
+
+    bool foundMatch = false;
+    for (unsigned i = 0; i < robotJointsListNames.size(); i++)
+    {
+        for (unsigned j = 0; j < humanJointsListName.size(); j++)
```

Comment:
probably you can simplify this part of the code using [`std::find`](https://en.cppreference.com/w/cpp/algorithm/find) and [`std::distance`](https://en.cppreference.com/w/cpp/iterator/distance)

Reply:
Using these standard commands are an option as well. However, I do not see any simplifications using them. Using them, we still will have one `for` loop, and inside that we should iterate using `find` and `if` condition, then using `distance` to check the element number. The inner loop is doing exactly the same! From the computational point of view, both approaches are linear in time.

Reply:
Yes it is fine for me